exercise.py

```diff
@@ -0,0 +1,13 @@
+
+# Test data [1,2, 4], [-1,-3], [1,2,3]
+def solution(A):
+    a_list = sorted(A)
+
+    if 1 not in a_list or a_list[-1] < 0:
+        return 1
+
+    for index in range(len(A)):
+        if index + 1 == len(A):
+            return a_list[index] + 1
+        elif a_list[index + 1] - a_list[index] > 1 and a_list[index] + 1 > 0:
+            return a_list[index] + 1
```

Comment:
what is the difference between 11 and 13?

Reply:
```
if (index + 1 == len(A)) or  (a_list[index + 1] - a_list[index] > 1 and a_list[index] + 1 > 0):
  return a_list[index] + 1
```

Reply:
a_list[index] + 1   - next or next_element

Reply:
a_list[index + 1] - a_list[index] > 1 - elements_difference
a_list[index] + 1 > 0 - next_is_positive?